lib/std/fs/file.zig

```diff
@@ -829,4 +829,165 @@ pub const File = struct {
     pub fn seekableStream(file: File) SeekableStream {
         return .{ .context = file };
     }
+
+    const range_off: windows.LARGE_INTEGER = 0;
```

Comment:
If those two fields are still needed you can slim down the structure by making them void on platforms that are not windows.

Reply:
Not sure I understand this suggestion - did you mistake these global variables for fields?

Reply:
Yep, taking them by pointer is what confused me.